packages/transaction-controller/src/types.ts

```diff
@@ -100,6 +102,76 @@ export interface Transaction {
   estimateGasError?: string;
 }
 
+/**
+ * @type TransactionReceipt
```

Comment:
I don't think we need this `type` parameter, but just a little description for the type like `Standard data concerning a transaction processed by the blockchain` for example.

Reply:
Done